uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/Service/DrupalUserServiceTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+use Civi\Test\HeadlessInterface;
+use Civi\Test\TransactionalInterface;
+use Civi\Test;
+use CRM_HRCore_Service_DrupalUserService as DrupalUserService;
+use CRM_HRCore_Service_DrupalRoleService as DrupalRoleService;
+
+/**
+ * @group headless
+ */
+class DrupalUserServiceTest extends \PHPUnit_Framework_TestCase implements HeadlessInterface, TransactionalInterface {
```

Comment:
Sure, I saw this. It would definitely be nice to have, but it was in the 1.7 branch only. Do you think I should add it? I thought about cherry picking, but the directory "uk.co.compucorp.civicrm.hrleaveandabsences" doesn't even exist now